src/http/api/webhook.controller.ts

```diff
@@ -72,9 +72,15 @@ export class WebhookController {
                 (await ctx.req.json()) as unknown,
             ).post.current;
         } catch (err) {
+            this.logger.error(
+                'Failed to parse post.published webhook payload: {error}',
+                { error: err instanceof Error ? err.message : 'Unknown error' },
+            );
```

Comment:
suggestion: might be useful to log the post.published payload too?

Reply:
Zod should give us an error like:

```
Failed to parse post.published webhook payload:
  [{"path":["post","current","uuid"],"message":"Invalid UUID"}]
```

I think this should be enough? (payloads could be large)